countries/burundi/pom.xml

```diff
@@ -60,91 +101,85 @@
             </goals>
             <configuration>
               <outputDirectory>
-                ${project.build.directory}/${project.artifactId}-${project.version}</outputDirectory>
+                ${project.build.directory}/${project.artifactId}-${project.version}/distro</outputDirectory>
```

Comment:
We should only override only distro folder and not the run/docker folder files from the ozone-uvl-distro in the ozone-uvl-burundi.